transport/x/grpc/handler.go

```diff
@@ -59,30 +60,42 @@ func (h *handler) handle(srv interface{}, serverStream grpc.ServerStream) error
 		return errInvalidGRPCStream
 	}
 
-	// Apply a unary request.
-	responseMD := metadata.New(nil)
-	response, err := h.handleBeforeErrorConversion(ctx, serverStream.RecvMsg, responseMD, stream.Method(), start)
-	err = handlerErrorToGRPCError(err, responseMD)
+	var requestData []byte
+	if err := serverStream.RecvMsg(&requestData); err != nil {
+		return err
+	}
+	requestBuffer := bufferpool.Get()
+	defer bufferpool.Put(requestBuffer)
+	// Buffers are documented to always return a nil error.
+	_, _ = requestBuffer.Write(requestData)
```

Comment:
nothing that can be handled?

Reply:
Like literally never returns error, it's a weird situation https://golang.org/pkg/bytes/#Buffer.Write